dataManipulation/generarTablasVariables.py

```diff
@@ -248,11 +249,18 @@ def retrieveIndicePrivacion(save=True,verbose=config.VERBOSE,yrs=[2016,2017,2018
     if len(keep):
         return([dfs[yr] for yr in keep])
     
-def prepare(df16,yr,predictors=False,indicePrivacion=False,verbose=config.VERBOSE):
+def prepare(df16,yr,predictors=False,indicePrivacion=False,verbose=config.VERBOSE,
+            excludeOSI=config.EXCLUDEOSI):
```

Comment:
Caution: This will throw error if EXCLUDEOSI is not in config (it is now a compulsory attribute). Address it in #13 .